pkg/test/istio.io/tasks/security/authz-http/authz-http_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+package tests
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/istio.io/examples"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+
+	"istio.io/istio/pkg/test/framework/components/istio"
+)
+
+var (
+	ist istio.Instance
+)
+
+const (
+	ns = "default"
+)
+
+func TestMain(m *testing.M) {
+	framework.NewSuite("authz-http", m).
+		SetupOnEnv(environment.Kube, istio.Setup(&ist, nil)).
+		RequireEnvironment(environment.Kube).
+		Run()
+}
+
+// TestAuthzHTTP simulates the task in https://www.istio.io/docs/tasks/security/authz-http/
+func TestAuthzHTTP(t *testing.T) {
+	ex := examples.New(t, "authz-http")
```

Comment:
From what I can tell, Examples are very lightweight.
So I decided to create one example for each top-level section of the original doc, in the same test case (`TestXYZ` test function).
So in your case, I'd create one `example` in each test method below.
That helps with having more localized logs in case of error.

Reply:
Done. Split to multiple examples.